packages/ariakit/src/collection/__examples__/collection/index.tsx

```diff
@@ -0,0 +1,20 @@
+import {
+  Collection,
+  CollectionItem,
+  useCollectionState,
+} from "ariakit/collection";
+import "./style.css";
+
+export default function Example() {
+  const collection = useCollectionState();
+
+  return (
+    <Collection state={collection} className="collection">
+      <CollectionItem className="collection-item">🍎 Apple</CollectionItem>
+      <CollectionItem className="collection-item">🍇 Grape</CollectionItem>
+      <CollectionItem className="collection-item">🍊 Orange</CollectionItem>
+      <CollectionItem className="collection-item">🍓 Strawberry</CollectionItem>
+      <CollectionItem className="collection-item">🍉 Watermelon</CollectionItem>
+    </Collection>
+  );
```

Comment:
Maybe something like this:
```suggestion
  const collection = useCollectionState();
  return (
    <Collection state={collection} className="collection">
      <div>Items count: {collection.items.length}</div>
      <CollectionItem>🍎 Apple</CollectionItem>
      <CollectionItem>🍇 Grape</CollectionItem>
      <CollectionItem>🍊 Orange</CollectionItem>
    </Collection>
  );
```

Reply:
Thank you, updated in https://github.com/reakit/reakit/pull/977/commits/4f909150cb671cf1738a9a279e2b5be691898b7f